File: adding.c
#include <stdio.h>

int main() {
    int num1, num2, num3, sum;

    // Input three numbers
    printf("Enter three numbers: ");
    scanf("%d %d %d", &num1, &num2, &num3);

    // Calculate the sum
    sum = num1 + num2 + num3;

    // Display the result
    printf("Sum of %d, %d, and %d is %d\n", num1, num2, num3, sum);

    return 0;
}

File: adding.py
def factorial(n):
    if n < 0:
        return "Factorial is not defined for negative numbers"
    elif n == 0 or n == 1:
        return 1
    else:
        result = 1
        for i in range(2, n + 1):
            result *= i
        return result

# Example usage:
number = int(input("Enter a number to find its factorial: "))
print(f"The factorial of {number} is {factorial(number)}")